docs/reference/licenses.md

```diff
@@ -6,31 +6,29 @@ description: "Licensing information for components of Camunda 8"
 
 ## Licensing
 
-This page contains licensing information for all components of Camunda 8.
+This page contains licensing information for all Components of Camunda 8 from version 8.6 and higher.
 
-### Zeebe
+### Zeebe, Operate, Tasklist, Identity, and Optimize
 
-Licenses and license information for Zeebe can be found in the Zeebe project [README](https://github.com/camunda/camunda#license).
+The source code for Zeebe, Operate, Tasklist, Identity, and Optimize is licensed to our users and customers under the [Camunda License v1](https://legal.camunda.com/licensing-and-other-legal-terms#camunda-license).
 
-### Operate, Tasklist, Identity, Optimize
+The compiled software of these components is distributed under a proprietary license, which can be either the [Camunda Self-Managed Non-Production license](https://legal.camunda.com/#self-managed-non-production-terms) or the Camunda Self-Managed Enterprise Edition license (a copy of which you obtain when you contact Camunda). To use the software in production, [purchase the Camunda Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/).
 
-Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Self-Managed Free Edition license](https://legal.camunda.com/licensing-and-other-legal-terms#legal-terms-camunda-platform-c8-self-managed-free-edition-and-beta-offerings) or the Camunda Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/). This licensing information also applies to web apps and APIs.
+### Camunda Connectors
```

Comment:
Do we need to qualify Connectors as "Camunda Connectors" anywhere else? This stood out to me.

Reply:
I don't think we need to qualify, as we are talking about Camudna here in general.